src/parsers/script-parser.js

```diff
@@ -202,6 +204,16 @@ const mapAttributesToOutput = (attribute) => {
         delete attribute.default;
     }
 
+    // enabledif is in a {}, so strip it out
```

Comment:
should it handle spaces? say `{ enabledif }` vs `{enabledif}`

Reply:
yeah from the editor side, it will handle both `@visibleif { condition }` and `@visibleif {condition}`